pages/linux/zenity.md

```diff
@@ -0,0 +1,23 @@
+# zenity
+
+> Display dialogs from the command line/shell scripts, and return user-inserted values or 1 if error.
```

Comment:
Split it into 2 lines like this - 

```
> Displays dialogs from command line / shell scripts.
> Returns user-inserted values or 1 in case of error.
```